cli/Dockerfile

```diff
@@ -44,6 +44,14 @@ RUN export DEBIAN_FRONTEND=noninteractive \
      && echo "deb [signed-by=/etc/apt/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list \
      && apt-get update \
      && apt-get install -y kubectl \
+     # Helm CLI
+     && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 \
+     && chmod 700 get_helm.sh \
+     && ./get_helm.sh \
+     # Kustomize CLI
+     && curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash \
+     # Kubelogin CLI
+     && az aks install-cli \
```

Comment:
I don't think we are using this Dockerfile anymore.

We are using a base image and devcontainer features.  It looks like we can use `ghcr.io/devcontainers/features/kubectl-helm-minikube:1` from https://containers.dev/features

Reply:
Some people are still using it.  I'll update the devcontainer in addition to the dockerfile.

Reply:
yeah.  The universal-img we have for azd includes all the tools supported by azd. It is convenient for getting started, but it gives projects more than what they need. For example, a python+nodejs project would ended up with dotnet, java, aks, etc.
The current size of the universal-azd-img is `> 1Gb`.  It takes several minutes to be created locally and during CI/CD.

By installing only what it is needed for you pipeline, you can save gh-action minutes (which are not unlimited for everyone).

I also talked to @danieljurek in the past about stop producing and pushing the universal-img to `mcr.microsoft.com/` as it is way to heavy to be continuously pushed.  I am not sure if we did stop that or not. :)

Reply:
We still push with updated bits on release https://mcr.microsoft.com/en-us/product/azure-dev-cli-apps/tags